### PR TITLE
restore use of bootstrap-responsive stylesheet without triggering hamburger menu

### PR DESCRIPTION
**Pull request by yoon**

### CODE REVIEW

File: app/assets/stylesheets/responsive.scss [hunk 1]
```diff
@@ -1,2 +1 @@
-@import "bootstrap-responsive";
 @import "bootstrap-responsive-overrides";
```
Comment (jhanggi) [inline app/assets/stylesheets/responsive.scss]: Don't need to do it now, but would it make sense to now pull whatever is in the `-overrides` file straight into this one?

Comment (yoon) [inline app/assets/stylesheets/responsive.scss]: Hmm. I was thinking about that... yeah, and I'll split them out again if needed.

Comment (jhanggi) [inline app/assets/stylesheets/responsive.scss]: Do you want to do that as part of this PR, or later? I'm good either way, just wanting to confirm if I should merge this or not.

Comment (yoon) [inline app/assets/stylesheets/responsive.scss]: Done, please merge.

**Pull request merged**